app/src/main/java/org/fossasia/openevent/app/core/sponsor/create/CreateSponsorView.java

```diff
@@ -8,4 +11,5 @@
 
     void dismiss();
 
+    void validate(TextInputLayout textInputLayout, Function<String, Boolean> function, String str);
```

Comment:
Why is there a need to add this?

Reply:
Sorry ,not needed. I'll remove it from interface and add a simple method